### PR TITLE
PBI-14: Delete comment

### LINKED ISSUE #41: PBI-14: Teacher & Creator Can delete comment of announcement in class
**Story Point: 5**
  
  - [x] Creator can delete the comment they wrote before
  - [ ] Show confirmation message in modal (?)

### PR DESCRIPTION
**Pull request by IlmaMannix**

### CODE REVIEW

File: src/resources/app/components/AnnouncementCard.vue [hunk 1]
```diff
@@ -85,7 +86,7 @@
 
 <script>
 import AnnouncementContent from "./AnnouncementContent";
-import {createComment} from "../../api/announcement/comment/createComment";
+import {createComment} from "../../api/comment";
```
Comment (AldiNFitrah) [inline src/resources/app/components/AnnouncementCard.vue:89]:
```suggestion
import { createComment } from "../../api/comment";
```
spaces between curly brackets

**Review by AldiNFitrah — APPROVED** [review]
that's it from me, great job!

File: src/resources/app/components/AnnouncementContent.vue [hunk 2]
```diff
@@ -160,6 +187,15 @@ export default {
       }
     },
 
+    async deleteComment(){
+      const {data, error, errorMessage} = await deleteComment(this.$props.id);
+      if (error) {
+        this.showSnackbar(errorMessage);
+      } else {
+        location.reload();
```
Comment (kanziebub) [inline src/resources/app/components/AnnouncementContent.vue:195]: this could be improved, but we can figure this out later

**Review by kanziebub — APPROVED** [review]
LGTM, thank you for your hard work!

**Pull request merged**